src/pages/Main/MainArtist.js

```diff
@@ -10,7 +12,12 @@ export default function MainArtist() {
           <p className="firstText">Meet our international artists</p>
           <p className="secondText">5월의 아티스트, Kris Gebhardt</p>
           <button className="textMore">
-            <p className="moreButton">작품 더보기</p>
+            <p
+              className="moreButton"
+              onClick={() => navigate("productDetail/:id")}
+            >
+              작품 더보기
+            </p>
```

Comment:
동적라우팅을 하기위해서는 
```suggestion
            <p
              className="moreButton"
              onClick={() => navigate(`productDetail/${실제 id값}`)}
            >
              작품 더보기
            </p>
```
으로 수정되어야 하겠네요!